src/main/java/com/kitaab/hisaab/ledger/exception/CustomExceptionHandler.java

```diff
@@ -60,4 +60,14 @@ private ResponseEntity<ErrorResponse> getErrorResponseResponseEntity(Exception e
                 traceId, spanId);
         return new ResponseEntity<>(message, HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(FlowBreakerException.class)
+    public ResponseEntity<ErrorResponse> handleApplicationException(Throwable ex) {
+        traceId = Objects.requireNonNull(tracer.currentSpan()).context().traceId();
+        spanId = Objects.requireNonNull(tracer.currentSpan()).context().spanId();
+        var exception = (FlowBreakerException) ex;
```

Comment:
we can directly use FlowBreakerException ex in method parameter . Why an extra conversion ?

Reply:
done the changes